virt/virt-4-11-release-notes.adoc

```diff
@@ -195,6 +195,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-11-known-issues"]
 == Known issues
 
+* When using the OVN-Kubernetes cluster network provider, if you create a large number of `NodePort` services, to access virtual machines (VMs) via SSH, it causes OVN-Kubernetes to crash. This is because RAM and CPU consumption of the `ovnkube-master` peaks; OVN-Kubernetes cannot scale to handle a large number of `NodePort` services. (link:https://issues.redhat.com/browse/OCPBUGS-1940[*OCPBUGS-1940*])
```

Comment:
I think the first sentence is a bit confusing and could use a rewrite. Also, if I understand the bug correctly, this might happen if you create a large number of node ports for any reason, so I'd suggest a generic intro sentence, followed by VM-related context. 

If you want to make the whole thing more concise, here's my suggested rewrite for you to take or leave. :)

```
* The OVN-Kubernetes cluster network provider crashes from peak RAM and CPU usage if you create a large number of `NodePort` services. This can happen if you use node ports to expose SSH access to a large number of VMs. (link:https://issues.redhat.com/browse/OCPBUGS-1940[*OCPBUGS-1940*])
** As a workaround, use the OpenShift SDN cluster network instead of OVN-Kubernetes.
```